meta-raspberrypi-mbl/conf/machine/raspberrypi3-mbl.conf

```diff
@@ -112,3 +112,7 @@ MBL_WKS_STORAGE_SIZE_MiB ?= "7168"
 MBL_WATCHDOG_TIMEOUT_SECS ?= "15"
 # Set the device file name for the hardware watchdog
 MBL_WATCHDOG_DEVICE_FILENAME ?= "/dev/watchdog"
+
+# Remove all conflicting meta-mbl-bsp-common linux-* recipes.
+BBMASK += "/meta-mbl/meta-mbl-bsp-common/recipes-kernel/linux/linux-fslc_%.bbappend"
+BBMASK += "/meta-mbl/meta-mbl-bsp-common/recipes-kernel/linux/linux-imx_%.bbappend"
```

Comment:
I believe these lines are not needed anymore.

Reply:
Removed.